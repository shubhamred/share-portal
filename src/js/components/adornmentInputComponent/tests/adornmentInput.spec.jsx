import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import AdornmentInput from '../adornmentInput';

configure({ adapter: new Adapter() });

describe('components/AdornmentInput', () => {
  it('Renders default options', () => {
    const component = shallow(
      <AdornmentInput />
    );
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
