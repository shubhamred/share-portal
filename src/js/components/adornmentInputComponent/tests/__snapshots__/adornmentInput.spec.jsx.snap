// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/AdornmentInput Renders default options 1`] = `
<WithStyles(ForwardRef(Grid))
  className="container"
  container={true}
  direction="row"
  item={true}
  justify="space-between"
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    sm={12}
    xs={12}
  >
    <WithStyles(ForwardRef(Grid))
      className="inputWrapper"
      item={true}
      sm={12}
      xs={12}
    >
      <WithStyles(ForwardRef(Input))
        className="input"
        disableUnderline={true}
        disabled={false}
        endAdornment={
          <ForwardRef(WithStyles(ForwardRef(InputAdornment)))
            position="end"
          >
            %
          </ForwardRef(WithStyles(ForwardRef(InputAdornment)))>
        }
        inputProps={
          Object {
            "max": "100",
            "min": "0",
            "step": ".1",
          }
        }
        onChange={[Function]}
        placeholder=""
        type="number"
        value=""
      />
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
